### DISCUSSION TITLE
Android 13 and Audi A1

### DISCUSSION THREAD

**Discussion by paulobreim** [AndreyPavlenko/Fermata#294] [Q&A]
I have a samsung S23 with Android 13 and my Android Auto is an AUDI A1.

When I start to see youtube videos, I can ear and control something, but there are no video images.
Same when play videos recorded into mobile.

Any idea?

Thank you 
paulo

**Comment by AndreyPavlenko** [comment]
You need to install the app with KingInstaller https://github.com/fcaronte/KingInstaller/releases
